rpms/s3/s3rpm.spec

```diff
@@ -366,6 +366,7 @@ rm -rf %{buildroot}
 /opt/seagate/cortx/s3/mini-prov/s3setup_prereqs.json
 /opt/seagate/cortx/s3/mini-prov/s3_prov_config.yaml
 /opt/seagate/cortx/s3/bin/s3prov_start_services.sh
+/opt/seagate/cortx/s3/bin/install_prereqs.sh
```

Comment:
This should not be packaged

Reply:
Even /opt/seagate/cortx/s3/reset/clean_openldap.sh should not be packaged

Reply:
Add copyright to scripts/provisioning/s3setup_prereqs.json

Reply:
About adding the copyright to scripts/provisioning/s3setup_prereqs.json, it is causing json failure as comments are not allowed,

Reply:
All the reviews in this PR are addressed except for the adding copyright info to json file. As of now JSON does not support comments and it is implement dependent. cortx-utils conf store also does not support json comments. However currently validator code path is not using confstore while parsing a json config file, so this code path can be updated to use conf store instead of using python json pkgs. I'll create a ticket for this work and add it to backlog.

Reply:
Note:
Looks like the current code in ./scripts/provisioning/postinstallcmd.py is already updated to use confstore to process the s3setup_prereqs.json. So no need to create a new ticket.